### DISCUSSION TITLE
ec_sys: failed to load when running MControlCenter

### DISCUSSION THREAD

**Discussion by kevkdev** [dmitry-s93/MControlCenter#188] [Q&A]
Platform: MSI-17k4 Raider

OS: Ubuntu 24 LTS
% uname -a
Linux TD-PC-1002 6.8.0-36-generic #36-Ubuntu SMP PREEMPT_DYNAMIC Mon Jun 10 10:49:14 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

Secure boot disabled in BIOS

% modprobe ec_sys write_support=1
% lsmod | grep ec_sys
ec_sys                 12288  0

Running the app, I get a pop-up.
Tried as both root and non-root with same results.

![image](https://github.com/dmitry-s93/MControlCenter/assets/911992/53924c9b-f290-448e-9ce4-3a6c3b66f83a)

Any recommendations to resolve this?

**Comment by mutchiko** [comment]
test the new version from the [releases page](https://github.com/dmitry-s93/MControlCenter/releases/tag/0.5.1)
